bridges/chains/chain-cumulus/Cargo.toml

```diff
@@ -19,6 +19,7 @@ workspace = true
 bp-messages = { workspace = true }
 bp-polkadot-core = { workspace = true }
 bp-runtime = { workspace = true }
+parachains-common = { workspace = true }
```

Comment:
I am just starting to think if we really need this module at the end, why not to move our stuff to the `parachains-common`?

@serban300 wdyt?

Reply:
Yes, yes, agree. I also mentioned this in https://github.com/paritytech/polkadot-sdk/pull/8107#pullrequestreview-2729400259

Reply:
sorry, sorry, I am blind :D, so yes, exactly as Serban said moving to parachains-common plus `mod bridging` (if needed)

> I don't know if we need both BlockWeights and BlockWeightsForAsyncBacking anymore.

Yes, we need both `BlockWeights` and `BlockWeightsForAsyncBacking`, because fellows is not switch to async backing and we are using still `BlockWeights` https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_kusama_config.rs#L397, only BHR/BHW use `BlockWeightsForAsyncBacking`

Reply:
Should I keep bp-bridge-hub-cumulus and just have it re-export everything from parachains-common? Just to avoid having to make a fellows fork

Reply:
I've moved the non-parameter-types constants over for now

Reply:
> Should I keep bp-bridge-hub-cumulus and just have it re-export everything from parachains-common? Just to avoid having to make a fellows fork

Why a fellows fork ?

> Thing is, chain-cumulus uses the other bp crates a fair amount. It seems a bit convoluted to have crates in /bridges as dependencies of stuff in parachains-common, which we are then using again in other parts of /bridges

You're right. I missed this. Yes, we can re-export than.

Reply:
> Why a fellows fork ?
I assume it would be necessary if we removed the bridge-hub-cumulus package entirely, to change the dependencies in fellows/runtimes

Reply:
Yes, but that's not a problem. Anyway since we decided to keep `bp-bridge-hub-cumulus` this is not applicable anymore.